gammapy/spectrum/models.py

```diff
@@ -43,10 +148,16 @@ class PowerLaw(SpectralModel):
     reference : float, `~astropy.units.Quantity` 
         :math:`E_0`
     """
-    def __init__(self, index, amplitude, reference):
+    DEFAULT_INDEX = 2.3 * u.Unit('')
+    DEFAULT_AMPLITUDE = 2.5 * 1e-12 * u.Unit('cm-2 s-1 TeV-1')
```

Comment:
Fortunately (for you) I spent quite some time yesterday chasing a bug that was introduced by a model having a default parameter where I didn't expect it. So I will remove the defaults.